### PR TITLE
Add toggles for backtick, empty, implicit array creation and short ternary

### LINKED ISSUE #248: Bug: disallowedShortTernary can not be disabled
rules can be individually disabled, but disallowedShortTernary is not in the configuration and can therefore not be disabled.

workaround in project phpstan.neon configuration file:

```
parameters:
    ignoreErrors:
        - identifier: ternary.shortNotAllowed
```

### PR DESCRIPTION
**Pull request by ruudk**
Fixes #248

### CODE REVIEW

File: rules.neon [hunk 1]
```diff
@@ -22,6 +22,10 @@ parameters:
 		uselessCast: %strictRules.allRules%
 		requireParentConstructorCall: %strictRules.allRules%
 		disallowedConstructs: %strictRules.allRules%
+		disallowedBacktick: %strictRules.disallowedConstructs%
```
Comment (ondrejmirtes) [inline rules.neon]: These lines should say `%strictRules.allRules%` as it does on all other lines.

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]
Please also do a separate option for `DynamicCallOnStaticMethodsCallableRule` and `DynamicCallOnStaticMethodsRule`, it's been on my todolist for 2.0 :)

File: rules.neon (hunk 1 above)
Comment (ruudk) [reply rules.neon]: Ah, this is going to be in 2.0, so we can break it.

**Comment by ruudk** [comment]
I made some changes, please have look.

**Review by ondrejmirtes — APPROVED** [review]

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**